examples/hello/.eslintrc

```diff
@@ -0,0 +1,19 @@
+{
+  "root": true,
+  "extends": [
+    "plugin:@dbos-inc/dbosRecommendedConfig"
+  ],
+  "plugins": [
+    "@dbos-inc"
+  ],
+  "env": {
+    "node": true,
+    "es6": true
+  },
+  "rules": {
+  },
+  "parser": "@typescript-eslint/parser",
+  "parserOptions": {
+    "project": "./tsconfig.json"
```

Comment:
This is still an irritation.  It won't lint files that aren't in this config.  But it makes you list the files to lint instead of using the config to establish that.  And, adding files to the config breaks something in the tests.

Reply:
Do you mean that we need to explicitly declare ` "include": ` in tsconfig? I think it's fine, because files aren't declared there will not be compiled (if I understand this correctly).

Reply:
So we have `include` in tsconfig.  Then, eslint takes files to lint, such as `eslint .`.  The `"project"` setting ties them together, but not the way I want - it complains if files presented to `eslint` are not covered by ts config (because then `tsc` doesn't generate types for them).  But I really wish `eslint` would just lint the same files as `tsc` sees automatically, rather than maintaining two lists that could get out of sync.

Also, putting files in `"includes"` is breaking something else.  I thought why not put `migrations` and `knexfile` in there, but this causes the tests to fail.

Continuing to investigate.